tests/todomanager-tests.js

```diff
@@ -0,0 +1,11 @@
+const assert = require('assert');
+const eoslime = require('eoslime').init('local');
```

Comment:
Don't use it like that -> instead use eoslime test

Reply:
fixed